src/app/backend-api.service.ts

```diff
@@ -2293,194 +1488,165 @@ export class BackendApiService {
-    return this.jwtPost(endpoint, BackendRoutes.RoutePathResendVerifyEmail, PublicKey, {
-      PublicKey,
-    });
-  }
-
-  VerifyEmail(endpoint: string, PublicKey: string, EmailHash: string): Observable<any> {
-    return this.post(endpoint, BackendRoutes.RoutePathVerifyEmail, {
-      PublicKey,
-      EmailHash,
-    });
-  }
-
-  GetUserMetadata(endpoint: string, PublicKeyBase58Check: string): Observable<GetUserMetadataResponse> {
-    return this.get(endpoint, BackendRoutes.RoutePathGetUserMetadata + "/" + PublicKeyBase58Check);
-  }
-
-  GetUsernameForPublicKey(endpoint: string, PublicKeyBase58Check: string): Observable<string> {
-    return this.get(endpoint, BackendRoutes.RoutePathGetUsernameForPublicKey + "/" + PublicKeyBase58Check);
-  }
-
-  GetPublicKeyForUsername(endpoint: string, Username: string): Observable<string> {
-    return this.get(endpoint, BackendRoutes.RoutePathGetPublicKeyForUsername + "/" + Username);
-  }
-
-  GetJumioStatusForPublicKey(endpoint: string, PublicKeyBase58Check: string): Observable<any> {
-    return this.jwtPost(endpoint, BackendRoutes.RoutePathGetJumioStatusForPublicKey, PublicKeyBase58Check, {
-      PublicKeyBase58Check,
-    });
+  ResendVerifyEmail(PublicKey: string) {
+    return from(
+      resendVerifyEmail({
+        PublicKey,
+      })
+    );
   }
 
-  SubmitETHTx(
-    endpoint: string,
-    PublicKeyBase58Check: string,
-    Tx: any,
-    ToSign: string[],
-    SignedHashes: string[]
-  ): Observable<any> {
-    return this.post(endpoint, BackendRoutes.RoutePathSubmitETHTx, {
-      PublicKeyBase58Check,
-      Tx,
-      ToSign,
-      SignedHashes,
-    });
+  VerifyEmail(PublicKey: string, EmailHash: string): Observable<any> {
+    return from(
+      verifyEmail({
+        PublicKey,
+        EmailHash,
+      })
+    );
   }
 
-  QueryETHRPC(endpoint: string, Method: string, Params: string[], PublicKeyBase58Check: string): Observable<any> {
-    return this.jwtPost(endpoint, BackendRoutes.RoutePathQueryETHRPC, PublicKeyBase58Check, {
-      Method,
-      Params,
-      PublicKeyBase58Check,
-    });
+  GetUserMetadata(PublicKeyBase58Check: string): Observable<GetUserMetadataResponse> {
+    return from(getUserMetadata({ PublicKeyBase58Check }, { nodeURI: "https://node.deso.org" }));
   }
 
-  AdminGetVerifiedUsers(endpoint: string, AdminPublicKey: string): Observable<any> {
-    return this.jwtPost(endpoint, BackendRoutes.RoutePathAdminGetVerifiedUsers, AdminPublicKey, {
-      AdminPublicKey,
-    });
+  AdminGetVerifiedUsers(): Observable<any> {
+    return from(adminGetVerifiedUsers());
   }
 
-  AdminGetUsernameVerificationAuditLogs(endpoint: string, AdminPublicKey: string, Username: string): Observable<any> {
-    return this.jwtPost(endpoint, BackendRoutes.RoutePathAdminGetUsernameVerificationAuditLogs, AdminPublicKey, {
-      AdminPublicKey,
-      Username,
-    });
+  AdminGetUsernameVerificationAuditLogs(Username: string): Observable<any> {
```

Comment:
do we want to keep such small methods that are just a middleware to the deso-protocol, or should we refactor to make this calls directly?

Reply:
If it's used more than once, having a helper function like this makes sense to me, but it may not make sense to keep it living here. We can discuss more during a sync.